packages/chap-lib/src/queuedRequest.txt

```diff
@@ -0,0 +1,442 @@
+// import { CancelablePromise, OpenAPIConfig } from './httpfunctions'
+// import { ApiRequestOptions } from './httpfunctions/core/ApiRequestOptions'
+// import { request as __request } from './httpfunctions/core/request'
+
```

Comment:
`txt` because this is just copied as a base, and we would get build errors if not ...